src/T2DMap.cpp

```diff
@@ -2055,7 +2059,8 @@ void T2DMap::paintEvent(QPaintEvent* e)
             // If one or more rooms are selected - make the text slightly orange.
             switch (selectionSize) {
             case 0:
-                infoText.append(tr("Room ID: %1 (Current) Position on Map: (%2,%3,%4)\n")
+                infoText.append(tr("Room ID: %1 (Current) Position on Map: (%2,%3,%4)\n",
```

Comment:
Because that `current` changes to `selected` or `%n selected` in other variants.  Perhaps we need some extra spaces after the closing `)` and the `Position` - they are really two pieces of information but presented on same line...

Reply:
So then.. `Selected room ID` seems obvious. Not so obvious for multiple rooms. Maybe like `%n rooms selected. Central room ID: etc`
I would restrain from multiple space characters, which may get lost in translation. Just use a hyphen or alike.